src/applications/gi/tests/components/CalculatorForm.unit.spec.jsx

```diff
@@ -52,11 +52,9 @@ const tree = mount(
 
 describe('<CalculatorForm>', () => {
   it('should display error message when beneficiary zip is less than 5 digits', () => {
-    const textBox = tree.find('#errorable-text-input-1');
```

Comment:
These are ids generated with `lodash/uniqueId`. I think the switch from requiring `lodash` to requiring `lodash/uniqueId` may have changed how lodash keeps track of what number its on.

In any case, we should not have an production code that relies on these ids, because we don't know them in advance, except when testing.

Reply:
FYI @zurbergram @dmccurdybah.